ptr.py

```diff
@@ -295,6 +295,27 @@ def _generate_pyre_cmd(
     return (str(pyre_exe), "--source-directory", str(module_dir), "check")
 
 
+def _parse_setup_params(setup_py: Path) -> Dict[str, Any]:
+    with setup_py.open("r", encoding="utf8") as sp:
+        setup_tree = ast.parse(sp.read())
+
+    LOG.debug("AST visiting {}".format(setup_py))
+    for node in ast.walk(setup_tree):
+        if isinstance(node, ast.Assign):
+            for target in node.targets:
+                # mypy error: "expr" has no attribute "id"
+                if target.id == "ptr_params":  # type: ignore
+                    LOG.debug("Found ptr_params in {}".format(setup_py))
+                    ptr_params = ast.literal_eval(node.value)
```

Comment:
We need to cast this to make Dict[str, Any] to make PyFI happy as this is what we expect from setup.py:
```
ptr_params = ast.literal_eval(node.value)
cast(Dict[str, Any], ptr_params)
```